single-host-httpclient/src/main/java/com/github/nhenneaux/resilienthttpclient/singlehostclient/ServerConfiguration.java

```diff
@@ -126,6 +126,11 @@ public Consumer<HttpRequest.Builder> getRequestTransformer() {
         return requestTransformer;
     }
 
+    // package-private for testing
```

Comment:
I think this comment contradicts the actual method's modifier

Reply:
Also, what's the point of having a getter for a final static field?

Reply:
The app is accessing **ServerConfiguration.DEFAULT_REQUEST_TRANSFORMER** defined in **single-host-httpclient** from both modules - **monitored-httpclient** & **single-host-httpclient**. So is the reason it was declared as public. 
Since you suggested not to have public, we made it private and made it available through static getter.
Let us know your suggestion.

Reply:
That was my question, not a suggestion ;-) Since you explained that it is used in multiple places, it is indeed should be `public` constant. I am sorry for this confusion.